_rules/inline-link-in-paragraph-has-distinguishable-style-on-hover-and-focus-548868.md

```diff
@@ -0,0 +1,340 @@
+---
+id: 548868
+name: Inline link has distinguishable style or content on hover and focus
```

Comment:
Reading the unrelated https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html I found:
> The Use of Color success criterion addresses changing only the color (hue) of an object or text without otherwise altering the object's form. The principle is that contrast ratio (the difference in brightness) can be used to distinguish text or graphics. For example, G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them is a technique to use a contrast ratio of 3:1 with surrounding text to distinguish links and controls. **In that case the Working Group regards a link color that meets the 3:1 contrast ratio relative to the non-linked text color as satisfying the Success Criteria 1.4.1 Use of color since it is relying on contrast ratio as well as color (hue) to convey that the text is a link.**

(emphasise mine)
I feel this indicates that G183 is indeed an overstatement and that a 3:1 colour contrast (and no other cue) is enough to pass 1.4.1.

…which could lead to further simplification of the rule by having "distinguishing style" include "distinguishing colors (with a 3:1 ratio)"…

Reply:
Updated the applicability to consider hue and the expectation to consider color (through distinguishable styles)